materialize-google-sheets/main.go

```diff
@@ -20,8 +20,8 @@ import (
 type driver struct{}
 
 type config struct {
-	CredentialsJSON string `json:"googleCredentials"`
-	SpreadsheetURL  string `json:"spreadsheetUrl"`
+	CredentialsJSON string `json:"googleCredentials" jsonschema:"title=Credentials,description=Service account JSON key to use as Application Default Credentials." jsonschema_extras:"secret=true,multiline=true"`
```

Comment:
I opted not to use schema generation in this connector, and directly implemented the schema below (in `Spec`).